website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -14,18 +14,22 @@ dbt "adapters" are responsible for _adapting_ dbt's functionality to a given dat
+2. If your making a adapter plugin from scratch can use the [dbt-database-adapter-scaffold](https://github.com/dbt-labs/dbt-database-adapter-scaffold) to trigger an interactive session which will allow to replace default naming, author information, versioning, and be used to augment other variables in the generation of your starting project.
 
-dbt comes equipped with a script which will automate a lot of the legwork in building a new adapter. This script will generate a standard folder structure, set up the various import dependencies and references, and create namespace packages so the plugin can interact with dbt. You can find this script in the dbt repo in dbt's [scripts/](https://github.com/dbt-labs/dbt-core/blob/HEAD/core/scripts/create_adapter_plugins.py) directory.
+    Example usage:
 
-Example usage:
+    ```
+    $ cookiecutter gh:dbt-labs/dbt-database-adapter-scaffold
+    ```
 
-```
-$ python create_adapter_plugins.py --sql --title-case=MyAdapter ./ myadapter
-```
-
-You will get a folder named 'myadapter' in the local directory, with some subfolders and files created. Your adapter will be named 'MyAdapter' in the generated code - without `--title-case=MyAdapter` it would be 'Myadapter'. You can set other flags to specify dependencies, author, and package information as well. If your adapter implements SQL's `information_schema` (or something similar enough) and supports a cursor() method on its connections, you may pass the `--sql` flag to derive from the SQLAdapter, which is much easier to implement than the BaseAdapter! Compare dbt's native BigQuery adapter with its SnowflakeAdapter to get an idea of the difference between the two.
+The generated boilerplate starting project will include basic adapter plugin file structure, examples of macros, high level method descriptions as seen below in this document.
 
-This rest of this guide will assume that a SQLAdapter is being used.
+One of the most important choices you will make during the cookiecutter generation will revolove around the field for `is_sql_adapter` which is a boolean used to correctly apply imports for either a `SQLAdapter` or `BaseAdapter`. Knowing which you will need requires a deeper knowledge of your selected database but a few good guides for the choice are.
+- Does your database have a complete SQL API? Can it perform tasks using SQL such as (creating schemas, droping schemas, querying an `information_schema` for metadata calls) if so it is more likely to be a SQLAdapter where you would say True to `is_sql_adapter`.
+- Most adapters do fall under SQL adapters which is why we chose it as the default `True` value.
+- It is very possible to build out a fully functional `BaseAdapter` just will require a little more ground work as it doesn't come with some prebuilt methods the `SQLAdapter` class provides. see `dbt-bigquery` as a good guide.
```

Comment:
```suggestion
- It is very possible to build out a fully functional `BaseAdapter`. This will require a little more ground work as it doesn't come with pre-built methods that the `SQLAdapter` class provides. See `dbt-bigquery` as a good guide.
```